docs/installation/control-plane-installation.md

```diff
@@ -5,46 +5,51 @@ This guide will show you how to install and configure the Multi-Cluster Gateway
 ## Prerequisites
 
 - A **hub cluster** running the OCM control plane (v0.11.0 or greater)
+- Addons enabled `clusteradm install hub-addon --names application-manager`
 - Any number of additional **spoke clusters** that have been configured as OCM [ManagedClusters](https://open-cluster-management.io/concepts/managedcluster/)
 - [Kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) (>= v1.14.0)
 - Either a pre-existing [cert-manager](https://cert-manager.io/)(>=v1.12.2) installation or the [Kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/) and [Helm](https://helm.sh/docs/intro/quickstart/#install-helm) CLIs
 - Amazon Web services (AWS) and or Google cloud provider (GCP) credentials. See the [DNS Provider](../dnspolicy/dns-provider.md) guide for obtaining these credentials.
 
 ## Configure OCM with RawFeedbackJsonString Feature Gate
 
-All OCM spoke clusters must be configured with the `RawFeedbackJsonString` feature gate enabled. This can be done in two ways:
+All OCM spoke clusters must be configured with the `RawFeedbackJsonString` feature gate enabled:
 
-1. When running the `clusteradm join` command that joins the spoke cluster to the hub:
+1. By patching each spoke cluster's `klusterlet` in an existing OCM install:
 
-    Get the `join` flags and token by running
-
-    ```bash
-    join=$(clusteradm get token --context kind-test-control-plane | grep -o 'join.*--cluster-name')
-    ```
-    
     ```bash
-    clusteradm $join --feature-gates=RawFeedbackJsonString=true
+    kubectl patch klusterlet klusterlet --type merge --patch '{"spec": {"workConfiguration": {"featureGates": [{"feature": "RawFeedbackJsonString", "mode": "Enable"}]}}}' --context <EACH_SPOKE_CLUSTER>
     ```
 
-2. By patching each spoke cluster's `klusterlet` in an existing OCM install:
+## Setup for hub commands
+Many of the commands in this document should be run in the context of your hub cluster.
+By configure HUB_CLUSTER which will be used in the commands:
 
-    ```bash
-    kubectl patch klusterlet klusterlet --type merge --patch '{"spec": {"workConfiguration": {"featureGates": [{"feature": "RawFeedbackJsonString", "mode": "Enable"}]}}}' --context <EACH_SPOKE_CLUSTER>
-    ```
+```bash
+export HUB_CLUSTER=<hub-cluster-name>
+```
+
+## Install Cert-Manager
```

Comment:
Moving this up here might no longer be necessary given the removal of the certissuer from the kustomize.

Reply:
It is prob still needed as if they want to use TLSPolicy they still need to install it